docs/core/examples/ftpclient.py

```diff
@@ -47,9 +46,9 @@ def fail(error):
 def showFiles(result, fileListProtocol):
     print('Processed file listing:')
     for file in fileListProtocol.files:
-        print('    %s: %d bytes, %s' \
-              % (file['filename'], file['size'], file['date']))
-    print('Total: %d files' % (len(fileListProtocol.files)))
+        print('    {}: {} bytes, {}'.format(
+              file['filename'], file['size'], file['date']))
+    print('Total: {} files'.format(len(fileListProtocol.files)))
 
 def showBuffer(result, bufferProtocol):
```

Comment:
It might be nice if this function decoded `bufferProtocol.buffer.getvalue()` as `charmap`, so that newlines in the buffer would render as newlines in the output.